apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -8,6 +8,99 @@ window.PLOrderBlocks = function (uuid, options) {
 
   let optionsElementId = '#order-blocks-options-' + uuid;
   let dropzoneElementId = '#order-blocks-dropzone-' + uuid;
+  let pressedKeys = {};
+
+  function initializeKeyboardHandling(optionsElementId, dropzoneElementId) {
+    function inDropzone(block) {
+      let parentArea = block.closest('.pl-order-blocks-connected-sortable');
+      return parentArea.classList.contains('dropzone');
+    }
+    function getIndentation(block) {
+      return Math.round(parseInt(block.style.marginLeft.replace('px', '') / TABWIDTH));
+    }
+    let fullContainer = document.querySelector('.pl-order-blocks-question-' + uuid);
+    let blocks = fullContainer.querySelectorAll('*.pl-order-block');
+
+    for (let block of blocks) {
+      block.addEventListener('click', () => {
+        block.focus();
+      });
```

Comment:
the code in this loop is nested pretty deep, it would be nice if we could avoid that. How about we define a function that takes a block and attaches all the event listeners to it, and the here we just have a `forEach` or a `map` over all the blocks?